lib/crier/notification.rb

```diff
@@ -2,7 +2,7 @@ module Crier
   class Notification < ActiveRecord::Base
     self.table_name = 'crier_notifications'
 
-    belongs_to :crier, :class_name => 'User'
+    belongs_to :crier, :class_name => 'User', :optional => true
```

Comment:
So this option ignored in older versions? It doesn't complain that it's never heard of it before?